www/js/util.js

```diff
@@ -1433,6 +1435,13 @@ function sendVideoUpdate() {
 
 /* chat */
 
+function stripImages(msg){
+    if (!USEROPTS.strip_image) {
+        return msg;
+    }
+    return msg.indexOf('<img') > -1 ? msg.replace(IMAGE_MATCH, "$1") : msg;
```

Comment:
I don't think the `indexOf` check optimizes anything here, both `indexOf` and the regex match are effectively going to do a linear scan until `<img` is located, since there is no branching in the regex up to that point.